### PR TITLE
Add optional parameter to fail if there is a file increase above the threshold

### PR DESCRIPTION
**Pull request by Elyahou**
Add optional parameter to fail if there is a file increase above the threshold (5%)

Related to https://github.com/fiverr/platform_sphinx/issues/1162

### CODE REVIEW

File: lib/diff/index.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ module.exports = (before, after) => {
 	const string = `${percentage > 0 ? '+' : ''}${percentage}%`;
 
 	if (percentage > COLOUR_THRESHOLD) {
+		global.sizeIncrease = 1;
```
Comment (odeadglaz) [inline lib/diff/index.js]: Why not just `true`?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "statistician",
-  "version": "1.0.0",
+  "version": "1.1.0-rc.failOnIncrease.1",
```
Comment (odeadglaz) [inline package.json]:
```suggestion
  "version": "1.1.0",
```

File: programs/github-pull-request/cli.js [hunk 3]
```diff
@@ -12,16 +12,18 @@ const getJSON = require('../../lib/getJSON');
  * @param  {String} options.bundle
  * @param  {String} options.file
  * @param  {String} options.projectName
+ * @param  {boolean} options.failOnSizeIncrease (optional) - fail the build if the size increase more than 5%
```
Comment (odeadglaz) [inline programs/github-pull-request/cli.js:15]: Can we change the limit increase to 10? 5 sounds a bit rough

Comment (Elyahou) [reply programs/github-pull-request/cli.js:15]: @raphaelboukara

Comment (raphaelboukara) [reply programs/github-pull-request/cli.js:15]: yes good idea

**Review by raphaelboukara — APPROVED** [review]

**Pull request merged**